### PR TITLE
Show submission history

### PR DESCRIPTION
**Pull request by almuntasir**
Update task-submission card

# Description

I added a code to list submission history activities. 

Fixes # (issue)

## Type of change

Please delete options that are not relevant.

- [ ] New feature (non-breaking change which adds functionality)

# How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration

# Checklist:

- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules

show submission history

If you have any questions, please contact @macite or @jakerenzella.

### CODE REVIEW

File: src/app/projects/states/dashboard/directives/task-dashboard/directives/task-submission-card/task-submission-card.tpl.html [hunk 1]
```diff
@@ -56,5 +56,26 @@ <h4>
...
+    <thead> 
+        <tr> 
+          <th> Comment </th>
+          <th> Type </th>
+          <th> Marker </th>
+          <th> Read Date </th>
+        </tr>                                             
+    </thead>
+    <tbody>
+      <tr ng-repeat="comment in task.comments">
```
Comment (macite) [inline src/app/projects/states/dashboard/directives/task-dashboard/directives/task-submission-card/task-submission-card.tpl.html:71]: What about non-status comments?

Comment (almuntasir) [reply src/app/projects/states/dashboard/directives/task-dashboard/directives/task-submission-card/task-submission-card.tpl.html:71]: Hi Andrew, 
Thank you for your comment, I will work on that on next trimester.

**Comment by jakerenzella** [comment]
Some other comments:

* Don't display the comment type, we should only show status comments here
* Don't label them as comments
* Don't use an unformatted date time in the user view
* Add the status icon
* Why are we adding the read date?

I don't really understand this entire change, what's the problem it's solving? If it were linking to the files at that point it could be useful but at this stage it isn't adding any functionality not included in the comment panel, even with the change Andrew has suggested I'm not sure this would be merged.

**Comment by macite** [comment]
As I mentioned there is another PR with this in it so I don’t know why this is still being worked on...

Please work on something else.

**Pull request closed without merging**